### PR TITLE
Add jax.random.poisson frontend

### LINKED ISSUE #13483: poisson

### PR DESCRIPTION
**Pull request by MuhammedAshraf2020**
closes #13483

### CODE REVIEW

File: ivy/functional/frontends/jax/random.py [hunk 1]
```diff
@@ -23,3 +23,15 @@ def uniform(key, shape=(), dtype=None, minval=0.0, maxval=1.0):
...
 def normal(key, shape=(), dtype=None):
     return ivy.random_normal(shape=shape, dtype=dtype, seed=ivy.to_scalar(key[1]))
+
+
+def _get_seed(key):
+    key1, key2 = int(key[0]), int(key[1])
+    return ivy.to_scalar(int("".join(map(str, [key1, key2]))))
+
+
+@handle_jax_dtype
```
Comment (zaeemansari70) [inline ivy/functional/frontends/jax/random.py:33]: It's failing for 'uint32' for tensorflow, kindly look into this

File: ivy/functional/frontends/jax/random.py [hunk 2]
```diff
@@ -23,3 +23,15 @@ def uniform(key, shape=(), dtype=None, minval=0.0, maxval=1.0):
 @to_ivy_arrays_and_back
 def normal(key, shape=(), dtype=None):
     return ivy.random_normal(shape=shape, dtype=dtype, seed=ivy.to_scalar(key[1]))
+
+
+def _get_seed(key):
```
Comment (zaeemansari70) [inline ivy/functional/frontends/jax/random.py:28]: As your previous PR is merged this is a duplicate function, remove this

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey @MuhammedAshraf2020! 
Thank you so much for contributing to ivy! 🙂

Your PR looks amazing ✨!
Just requested a few changes and left comments on them which needs to be addressed! 🙂

Do let me know if you have any other questions!
Thanks!

**Comment by MuhammedAshraf2020** [comment]
@zaeemansari70, 
can you check now please ?

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Your PR looks good, can you fix the merge conflicts?
There's also 1 lint error which you'll have to fix.
I'll merge your PR as soon you make these changes. 😄 

Thanks

**Comment by MuhammedAshraf2020** [comment]
@sherry30 
can you merge now please ?

**Review by sherry30 — APPROVED** [review]
Looks good,
Thanks a lot for the contribution 😄

**Pull request merged**